src/app/component/layout/components/footer/footer.component.scss

```diff
@@ -1,27 +1,22 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat|Open+Sans|Roboto&display=swap');
 
-$generalFont : OpenSans, sans-serif;
+$generalFont : 'Open Sans';
 
 .main-background-image {
-  background: url('/assets/img/icon/footer/background-image-footer.svg');
-  background-repeat: no-repeat;
-  background-position: center;
+  background: url('src/assets/img/icon/footer/background-image-footer.svg') no-repeat center;
```

Comment:
why did you change here?

Reply:
> why did you change here?

Is it better to leave 3 styles, if you can write them in one? It is something similar to margin etc: instead of writing margin-top, margin-bottom... you may write only margin

Reply:
it's not about this, but about:
```css
background: url('src/assets/img/icon/footer/background-image-footer.svg');
```
vs
```css
background: url('/assets/img/icon/footer/background-image-footer.svg');
```